src/node/audio_buffer_source.rs

```diff
@@ -55,7 +55,7 @@ struct PlaybackInfo {
     k: f32,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Copy)]
 struct LoopState {
```

Comment:
To continue housekeeping, I think we could just remove this struct altogether now that all updates are made through message passing and that it is not used anymore to save some `Arc`

Reply:
Ah, that's where it is coming from. But I do think it improves the readability slightly (makes the Renderer struct less daunting)